plugins/modules/netbox_cluster.py

```diff
@@ -159,6 +162,7 @@ def main():
                     cluster_type=dict(required=False, type="raw"),
                     cluster_group=dict(required=False, type="raw"),
                     site=dict(required=False, type="raw"),
+                    tenant=dict(required=False,type="str"),
                     comments=dict(required=False, type="str"),
```

Comment:
Could you add `tenant_group` here as well as that looks like an available field?

Also, could you add a test to the integration testing to create or update a cluster with a tenant?